main.fmf

```diff
@@ -8,13 +8,13 @@ require:
 # shared by all tests
 recommend:
   - python3
-  - python36
+  - python3.11
   - python3-requests
-  - python36-requests
+  - python3.11-requests
   - python3-pyyaml
-  - python36-pyyaml
-  - python3-rpm
-  - python36-rpm
+  - python3.11-pyyaml
+  - python3.11-urllib3
+  - python3-dnf
```

Comment:
Consider moving these under `require:` (or `require+:`) below, to the `adjust` secions.

Reply:
I think this won't be possible through `adjust` because `rhel-9` matches also all `rhel-9.Y` versions and, on the other hand, something like `distro == rhel-9 and distro ~!= rhel-9.0` won't match `distro=rhel-9` context (needed because of runcontest). So I just used `recommend+` instead of `require+` and the way it is now should properly set `CONTEST_PYTHON` variable based on distro.

Reply:
Sure, either way works. Thanks for trying!